packages/core/src/components/EstimatedFee/EstimatedFee.tsx

```diff
@@ -173,16 +195,50 @@ export default function EstimatedFee(props: FeeProps) {
     if (ests) {
       const estimateList = ests.estimates;
       const targetTimes = ests.targetTimes;
-      if (estimateList[0] == 0 && estimateList[1] == 0 && estimateList[2] == 0) {
-        setInputType("classic");
-      }
-      const est0 = formatEst(estimateList[0], multiplier, locale);
-      const est1 = formatEst(estimateList[1], multiplier, locale);
-      const est2 = formatEst(estimateList[2], multiplier, locale);
-      setEstList(current => []);
-      setEstList(current => [...current, { time: targetTimes[0] / 60, timeText: "Likely in " + targetTimes[0] + " seconds", estimate: est0 }]);
-      setEstList(current => [...current, { time: targetTimes[1] / 60, timeText: "Likely in " + (targetTimes[1] / 60) + " minutes", estimate: est1 }]);
-      setEstList(current => [...current, { time: targetTimes[2] / 60, timeText: "Likely over " + (targetTimes[2] / 60) + " minutes", estimate: est2 }]);
+      // if (
+      //   estimateList[0] == 0 &&
+      //   estimateList[1] == 0 &&
+      //   estimateList[2] == 0
+      // ) {
+      //   //setInputType('classic');
+      // }
+      const est0 =
+        estimateList[0] === 0
+          ? formatEst(6_000_000, 1, locale)
+          : formatEst(estimateList[0], multiplier, locale);
+      const est1 =
+        estimateList[1] === 0
+          ? formatEst(5_000_000, 1, locale)
+          : formatEst(estimateList[1], multiplier, locale);
+      const est2 =
+        estimateList[2] === 0
+          ? formatEst(0, 1, locale)
+          : formatEst(estimateList[2], multiplier, locale);
```

Comment:
Defining some default values when the fee estimator returns 0. 6m mojos for 1 min, 5m mojos for 2 mins, 0 mojos for > 2 mins